src/service/data_products/genome_attributes.py

```diff
@@ -220,6 +225,76 @@ def _remove_keys(doc):
     return doc
 
 
+##########################
+# Filter handling code
+##########################
+
+# might want to move some of this into a shared file at some point, might be reusable for other
+# DPs
+# Not sure about the meta call, may need to abstract that out somehow
+
+_FILTER_PREFIX = "filter_"
+
+
+async def _get_filters(
+    r: Request,
+    coll: models.ActiveCollection,
+    load_ver: str,
+    load_ver_override: bool,
+    skip: int,
+    limit: int,
+) -> FilterSet:
+    # TODO FILTER docs for filters
+    # TODO FILTER doc how to change / update the arangosearch view: CLI -> config in API
+    # TODO FILTER Add param for OR vs AND filters
+    # TODO FILTER count
+    # TODO FILTER match & selection
+    # TODO FILTER sort
+    filter_query = {}
+    for q in r.query_params.keys():
```

Comment:
I am curious how did you test your code and what are inside r.query_params dict?

Reply:
> how did you test your code

Fired up the server and used requests to exercise the endpoint

> what are inside r.query_params dict?

https://www.starlette.io/requests/#query-parameters